src/Symfony/Component/Notifier/Bridge/SimpleTextin/SimpleTextinTransport.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\SimpleTextin;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class SimpleTextinTransport extends AbstractTransport
+{
+    protected const HOST = 'api-app2.simpletexting.com';
+
+    public function __construct(
+        #[\SensitiveParameter] private readonly string $apiKey,
+        private readonly ?string $from = null,
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null
+    ) {
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        $queryParameters = [];
+        if ($this->from) {
+            $queryParameters['from'] = $this->from;
+        }
+
+        return sprintf('simpletextin://%s', $this->getEndpoint()).($queryParameters ? '?'.http_build_query($queryParameters) : null);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage;
```

Comment:
Please check for the corresponding Options class too, like in other bridges. Thanks